server/startup/migrations/v229.js

```diff
@@ -0,0 +1,36 @@
+import { Settings } from '../../../app/models/server';
+import { Migrations } from '../../../app/migrations';
+
+Migrations.add({
+	version: 229,
+	up() {
+		const oldNamesValidationSetting = Settings.findById(
+			'UTF8_Names_Validation',
+		);
+		const oldNamesValidationSettingValue =			oldNamesValidationSetting.value || '[0-9a-zA-Z-_.]+';
```

Comment:
better to be safe here to avoid locking the migrations
```suggestion
		const oldNamesValidationSettingValue = oldNamesValidationSetting?.value || '[0-9a-zA-Z-_.]+';
```

Reply:
added the change!